package.json

```
@@ -21,16 +21,17 @@
     "start-emulator-persist": "gcloud beta emulators datastore start --host-port=:15606 --use-firestore-in-datastore-mode",
     "start-emulator": "gcloud beta emulators datastore start --host-port=:15606 --no-store-on-disk --use-firestore-in-datastore-mode",
     "stop-emulator": "curl -X POST 'http://localhost:15606/shutdown'; pkill -f \"CloudDatastore.jar\"",
-    "build": "gulp",
-    "watch": "gulp watch",
+    "build": "npm run tsc-clean && tsc && gulp",
```

I'd like to be able to take advantage of the incremental build, and also not to take the time to run `tsc` twice every time we have to start the server. The need to clean each time doesn't need to be fixed in this PR, but keep it on your radar as a future optimization.
```suggestion
    "build": "npm run tsc-clean && gulp",
```